src/main/java/Storage.java

```diff
@@ -0,0 +1,133 @@
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A class that handles create, read and write file operations.
+ */
+public class Storage {
+    private final File dataFile;

```

Comment:
Shouldn't the constant variables name be all uppercase using underscore to separate words?